week06_policy_based/a2c-optional.ipynb

```diff
@@ -6,35 +6,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-     "import sys\n",
-     "if 'google.colab' in sys.modules:\n",
-     "    import os\n",
-     "\n",
-     "    # Undo the recent upgrade of importlib-metadata\n",
-     "    ## Thanks to @Mxhiu: https://stackoverflow.com/a/73932581/14473118\n",
-     "    os.system(\"pip install -q importlib-metadata==4.13.0\")\n",
-     "    os.system('apt-get install -y xvfb')\n",
-     "    os.system('wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/xvfb -O ../xvfb')\n",
-     "    # Download `atari_wrappers.py` script from github\n",
-     "    os.system(\"wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/week06_policy_based/atari_wrappers.py\")\n",
-     "    # Download `env_batch.py` script from github    \n",
-     "    os.system(\"wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/week06_policy_based/env_batch.py\")\n",
-     "    # Download `runners.py` script from github    \n",
-     "    os.system(\"wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/week06_policy_based/runners.py\")\n",
-     "    \n",
-     "    os.system('apt-get install -y python-opengl ffmpeg')\n",
-     "    os.system('pip install pyglet==1.2.4')\n",
-     "    # Update the gym environment to be compatible with the atari environment.\n",
-     "    os.system('pip install gym[atari,accept-rom-license]==0.21.0')\n",
-     "    os.system('python -m pip install -U pygame --user')\n",
-     "\n",
-     "    print('setup complete')\n",
-     "\n",
-     "# XVFB will be launched if you run on a server\n",
-     "import os\n",
-     "if type(os.environ.get(\"DISPLAY\")) is not str or len(os.environ.get(\"DISPLAY\")) == 0:\n",
-     "    !bash ../xvfb start\n",
-     "    os.environ['DISPLAY'] = ':1'"
+    "import sys, os\n",
+    "if 'google.colab' in sys.modules and not os.path.exists('.setup_complete'):\n",
+    "    # Install xvfb and our launcher script for it\n",
+    "    !apt-get install -y xvfb\n",
+    "    !wget -q https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/xvfb -O ../xvfb\n",
+    "\n",
+    "    # Download dependencies from Github\n",
+    "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/week06_policy_based/atari_wrappers.py\n",
+    "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/week06_policy_based/env_batch.py\n",
+    "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/week06_policy_based/runners.py\n",
+    "    \n",
+    "    # Update the gym environment to be compatible with the Atari environment\n",
+    "    !pip install gym[atari,accept-rom-license]\n",
```

Comment:
It seems that this (and the installation of `xvfb` above) suffices with the latest Colab.